packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -96,21 +98,20 @@ function useMovingAnimation(
 
 			return;
 		}
-		ref.current.style.transform = 'none';
+
+		ref.current.style.left = '';
+		ref.current.style.top = '';
```

Comment:
above you reset the values with 0 and here with ''. Any particular reason?

Reply:
No reason, I think ended up thinking an empty string might be better since that's what you get if no style is set. I'll fix the other lines.